simp_le.py

```diff
@@ -848,8 +874,7 @@ def test(args):
 
 def check_plugins_persist_all(ioplugins):
     """Do plugins cover all components (key/cert/chain)?"""
-    persisted = IOPlugin.Data(
-        account_key=False, key=False, cert=False, chain=False)
+    persisted = IOPlugin.Data()
```

Comment:
That's clearly wrong, as `IOPlugin.Data()` would contain `None` instead of `False`